src/tasks/dailyItems.ts

```diff
@@ -148,6 +148,21 @@ export const DailyItemTasks: Task[] = [
         }
       },
     },
+    {
+      name: "2002 Mr. Store",
+      // eslint-disable-next-line libram/verify-constants
+      ready: () => have($item`2002 Mr. Store Catalog`),
+      completed: () =>
+        get("availableMrStore2002Credits", 0) === 0 && get("_2002MrStoreCreditsCollected", false),
+      do: (): void => {
+        const bestItem = maxBy(
+          // eslint-disable-next-line libram/verify-constants
+          $items`Spooky VHS Tape, "I survived Y2K" T-Shirt, Letter from Carrie Bradshaw, pro skateboard, Mr. Accessaturday, Meat Butler, Loathing Idol Microphone, Charter: Nellyville, Manual of Secret Door Detection, Flash Liquidizer Ultra Dousing Accessory, Amulet of Perpetual Darkness, Giant black monolith`,
```

Comment:
```suggestion
          $items`Spooky VHS Tape, "I survived Y2K" T-Shirt, Letter from Carrie Bradshaw, pro skateboard, Mr. Accessaturday, Meat Butler, Loathing Idol Microphone, Charter: Nellyville, Manual of Secret Door Detection, Flash Liquidizer Ultra Dousing Accessory, Amulet of Perpetual Darkness, Giant black monolith, Crimbo cookie sheet`,
```

Reply:
Apparently I'm a cookie hater, I used phreds idea for sellsItem to fix it